src/it/scala/tech/cryptonomic/conseil/client/EndpointClient.scala

```diff
@@ -190,21 +192,95 @@ object DataEndpointsClientProbe {
         uri = endpoint,
         `Content-Type`(MediaType.application.json),
         Accept(MediaType.application.json),
-        Header("apiKey", "hooman")
+        Header("apiKey", apiKey)
       )
       client.expect[Json](req).attempt.map {
-        case Right(json) =>
-          Either.cond(
-            json == expected,
-            right = json,
-            left = new Exception(
-              s"Failed to match on $endpoint: expected \n${expected.spaces2} \n but found \n ${json.spaces2}"
-            )
-          )
+        case Right(received) =>
+          Diff[Json].apply(ignoreOperationId(received), ignoreOperationId(expected)) match {
+            case Identical(value) =>
+              value.asRight[Throwable]
+            case different =>
+              new Exception(
+                s"Failed to match on $endpoint: diff result is \n${different.show}"
+              ).asLeft[Json]
+          }
+        case left => left
+      }
+    }
+
+  }
+
+  def bakingRightsQueryEndpoint = {
```

Comment:
Notes: Testing bakers' rights in the integration to make sure that the code redesign will not break the concurrent operations.

It's a safety measure needed to handle the fact that the scheduling is now done within cats-effect's pure system.